ratis-server/src/test/java/org/apache/ratis/MiniRaftCluster.java

```diff
@@ -269,6 +283,19 @@ public MiniRaftCluster initServers() {
     return this;
   }
 
+  private void initDataStreamServer() {
+    LOG.info("Setting up data stream servers");
+    for (RaftServerProxy serverProxy : servers.values()) {
+      serverProxy.getDataStreamServerRpc().addRaftPeers(getOtherRaftPeers(serverProxy.getId()));
+    }
+  }
+
+  private Collection<RaftPeer> getOtherRaftPeers(RaftPeerId id) {
+    HashMap peerMap = new HashMap<>(peers);
+    peerMap.remove(id);
+    return peerMap.values();
```

Comment:
Replace it with the following.
```suggestion
    return peers.values().stream().filter(p -> !p.getId().equals(id)).collect(Collectors.toList());
```